components/markets/BuyFullSetForm.tsx

```diff
@@ -26,31 +29,34 @@ const BuyFullSetForm = ({
   const [sdk] = useSdkv2();
   const wallet = useWallet();
   const notificationStore = useNotifications();
-  const { data: market } = useMarket({ marketId });
-  const { data: saturatedMarket } = useSaturatedMarket(market ?? undefined);
+
   const { data: pool } = usePool({ marketId: marketId });
+
   const baseAssetId = pool?.baseAsset
     ? parseAssetId(pool.baseAsset).unrightOr(undefined)
     : undefined;
   const { data: metadata } = useAssetMetadata(baseAssetId);
 
-  const colors = market?.categories
-    ? calcMarketColors(marketId, market.categories.length)
-    : [];
-
   const [amount, setAmount] = useState<string>("0");
   const [maxTokenSet, setMaxTokenSet] = useState<Decimal>(new Decimal(0));
 
-  const { data: baseAssetBalance } = useBalance(
-    wallet.getActiveSigner()?.address,
-    baseAssetId,
-  );
+  const extrinsicBase = wallet.activeAccount?.address
+    ? sdk
+        ?.asRpc()
+        .api.tx.balances.transfer(wallet.activeAccount?.address, ZTG.toFixed(0))
```

Comment:
@Robiquet When doing transfers using a proxy should we transfer on behalf of `wallet.readlAddress` ?